config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```diff
@@ -85,19 +85,19 @@ spec:
     - name: authorized_keys
       type: publicKeySSH
       privateKeyPath: ["amster", "id_rsa"]
-  - name: truststore
-    keys:
-    - name: cacerts
-      type: truststore
-      keyStorePath: ["truststore", "cacerts-keypin"]
-      storePassPath: ["truststore", "cacerts-keypin"]
-      keystoreAliases:
-        - alias: ca-cert
-          type: caCopy
-          caPath: ["platform-ca-public", "public-key"]
-    - name: cacerts-keypin
-      type: literal
-      value: changeit
+  # - name: truststore
```

Comment:
why is this commented out?

Reply:
truststore breaks the secret manager. See issue #48
This should be temporary

Reply:
I might know why it breaks it, see comment in https://github.com/ForgeRock/secret-agent/pull/45

Reply:
yeah. I will be trying those fixes as part of issue #48 . For now, I think those fixes are our of scope of this PR